### DISCUSSION TITLE
EPEL 9 cobbler and cobbler-web RPM Packages

### DISCUSSION THREAD

**Discussion by manoj0772** [cobbler/cobbler#3215] [General]
Can someone please tell where do i find cobbler and cobbler-web RPM packages for EPEL 9. I am not seeing anything in EPEL 9. For EPEL 8 i see them here https://dl.fedoraproject.org/pub/epel/8/Modular/x86_64/Packages/c/

https://dl.fedoraproject.org/pub/epel/9/ - Nothing here.

**Comment by SchoolGuy** [comment]
CC @opoplawski

**Comment by opoplawski** [comment]
See https://bugzilla.redhat.com/show_bug.cgi?id=2052575  I think we still have some missing dependencies.  Looks like pymongo.  I suppose we could drop the mongo support - not sure how many people use that and I think that's being remove from cobbler anyway?

There is also the question of what version of cobbler to build.  I was going to build 3.3.3, which will not provide cobbler-web.

EPEL has decided to not provide modules yet for EL9 so I don't have the option to provide multiple module streams like in EL8.

**Reply by SchoolGuy** [reply]
I recently habe readded tests to provide a stable experience for MongoDB. Thus it is an optional dependency. However these tests (and the experience coming along with it), will be only included with Cobbler 3.4.0 and later.

**Comment by zparihar** [comment]
You should find them in the EPEL for AlmaLinux and Rocky Linux 9

**Comment by opoplawski** [comment]
To be clear - cobbler-3.3.3 is in EPEL9.  There is no cobbler-web for cobbler > 3.2.
